spacepy/lib.py

```diff
@@ -99,7 +99,7 @@ def load_lib():
     """
     libdir = os.path.dirname(os.path.abspath(__file__))
     if sys.platform == 'win32':
-        libnames = ['spacepy.dll']
+        libnames = ['spacepy.dll', 'libspacepy.dll.a']
```

Comment:
Does list order imply priority order? And if so, is there a possibility that an old dll could be found (say, if spacepy was installed without cleanup of a previous install)?

Reply:
They are checked in that order. It doesn't look like either setup.py or pip removes files if installing over an existing version, which doesn't make me happy.

I guess we _probably_ should put a version symbol in there to be sure it matches (although we're not doing that for anything else and I can see it maybe possibly being a problem for, say, irbem.) I don't necessarily want to jump to that.

How about I 1. do like with pycdf, and test-load each one found, so if there are two and one is (say) wrong architecture, it just won't load and move on to the next; 2. put the dll.a first, because even though _in general_ I'm not convinced that's more likely to be the newer one, _in this particular case_ it probably will be.

And then think about this issue later.